src/main/java/nl/windesheim/codeparser/analyzers/abstractfactory/AbstractFactoryAnalyzer.java

```diff
@@ -28,14 +27,16 @@
         }
 
         InterfaceFactoryFinder interfaceFinder = new InterfaceFactoryFinder();
-        AbstractFactoryFinder abstractFinder = new AbstractFactoryFinder();
 
         List<ClassOrInterfaceDeclaration> declarations = this.findDeclarations(files);
 
         List<ClassOrInterfaceDeclaration> factoryClasses = new ArrayList<>();
 
         factoryClasses.addAll(interfaceFinder.find(declarations));
-        factoryClasses.addAll(abstractFinder.find(declarations));
+
+        // Removing the abstract factory finder since it was deemed not necessary by our product owner.
+//        AbstractFactoryFinder abstractFinder   = new AbstractFactoryFinder();
```

Comment:
Als het er uitzag van de PO kan je het dan niet beter in zijn geheel verwijderen?

Reply:
Daar heb je wel een goed punt. Gone!